webui/src/pages/auth/users/user/policies.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 import {AuthLayout} from "../../../../lib/components/auth/layout";
-import {UserHeader} from "../../../../lib/components/auth/nav";
+import { UserHeaderWithContext } from "./userHeaderWithContext";
```

Comment:
Align the format "{ something }" vs "{something}" - maybe we should have something like prettier

Reply:
Yes, we should and we will :)